source/plugin/data/index.rst

```diff
@@ -0,0 +1,78 @@
+============
+The Data API
+============
+
+Overview
+========
+
+The unified Data API aims to provide a consistent way of accessing and modifying data of any kind. Where other
```

Comment:
Brain dump on defining "data":

#### Data
Any type of data that is consistently synchronized from the game server to the clients. If it can be changed and is synchronized to the connected clients, that is Data that the Data API should cover. Properties on the other hand, are data that can *not* be synchronized from the server to the client throughout the lifetime of the game, without requiring modifications to both the server and the client. Properties are explained in "insert article on properties here".